api.planx.uk/send/uniform.ts

```diff
@@ -233,11 +243,17 @@ export async function createUniformSubmissionZip(sessionId: string) {
   const overviewPath = path.join(tmpDir, "Overview.htm");
   const overviewFile = fs.createWriteStream(overviewPath);
   const overviewStream = generateHTMLOverviewStream(
-    csv as PlanXExportData[]
+    responses as PlanXExportData[]
   ).pipe(overviewFile);
-  await resolveStream(overviewStream);
-  zip.addLocalFile(overviewPath);
-  deleteFile(overviewPath);
+  await addToZip(overviewStream, overviewPath);
+
+  // generate and add an HTML overview document for the submission to zip
+  const redactedOverviewPath = path.join(tmpDir, "RedactedOverview.htm");
```

Comment:
Looking back at this, it is awkward to have to maintain this list of files in two locations and far from ideal.

I believe the reason we didn't pass in the list of files from here directly into `addOneAppXMLToZip()` was so that - 
 - we can generate a payload **just** off a session id
 - we can generate the XML outside the scope of a submission (e.g. the admin endpoint for debugging)
 
 I think it would be possible to generate an object which could be iterated over in this file, and in `getGeneratedFiles()`, which described the files we send to Uniform in such a way we don't need to repeat ourselves.
 
Outside the scope of this PR and it can totally be picked up later but thought it was worth discussing whilst looking at this.